client/src/components/Shared/Form/index.js

```diff
@@ -182,20 +183,13 @@ class Form extends Component {
               />
             </SelectStyle>
 
-            <SelectStyle>
-              <CreatableSelect
-                value={selectedBrand}
-                isDisabled={false}
-                isLoading={false}
-                isClearable={false}
-                isRtl={false}
-                isSearchable
-                name="Brand"
-                options={brands}
-                onChange={handleChange}
-                styles={this.errorStyle(isErrorBrand)}
-              />
-            </SelectStyle>
+            <StyledInputBrand
+              onChange={toggleOpen}
```

Comment:
I don't think we need this toggleOpen anymore.

Reply:
need